inginious/frontend/static/js/task_dispensers.js

```diff
@@ -101,9 +101,9 @@ function dispenser_util_click_modal_task(task) {
     input.attr("checked", !input.attr("checked"));
 }
 
-function dispenser_util_add_tasks_to_section(button) {
+function dispenser_util_add_tasks_to_section(button, checker) {
     task_id= $("#new_task_id").val();
-    if(!task_id.match(/^[a-zA-Z0-9_\-]+$/)){
+    if(checker && !task_id.match(/^[a-zA-Z0-9_\-]+$/)){
```

Comment:
Why not simply use the value of `existing_task` variable to define if you need to do this verification?

Reply:
Damn you are probably right!
I will do this instead :)